src/transactions/feed/SwapFeedItem.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import { HomeEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import TokenDisplay from 'src/components/TokenDisplay'
+import Touchable from 'src/components/Touchable'
+import SwapIcon from 'src/icons/SwapIcon'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import colors from 'src/styles/colors'
+import fontStyles from 'src/styles/fonts'
+import variables from 'src/styles/variables'
+import { useTokenInfo } from 'src/tokens/hooks'
+import { TokenExchange } from 'src/transactions/types'
+
+interface Props {
+  exchange: TokenExchange
+}
+
+function SwapFeedItem({ exchange }: Props) {
+  const { t } = useTranslation()
+  const incomingTokenInfo = useTokenInfo(exchange.inAmount.tokenAddress)
+  const outgoingTokenInfo = useTokenInfo(exchange.outAmount.tokenAddress)
+
+  const openTransferDetails = () => {
+    navigate(Screens.TransactionDetailsScreen, { transaction: exchange })
+    ValoraAnalytics.track(HomeEvents.transaction_feed_item_select)
+  }
+
+  return (
+    <Touchable disabled={false} onPress={openTransferDetails}>
+      <View style={styles.container}>
+        <View style={styles.iconContainer}>
```

Comment:
i think you don't need this `View`?

Reply:
The `Touchable` from 'src/components/Touchable' only allows one child component.

Reply:
yeah but then you can use a fragment

Reply:
I removed `iconContainer` view because it didn't affect anything when the font is small or the font is extra large.